src/account.ts

```diff
@@ -122,6 +122,9 @@ export class AccountError extends Error {
     static missingTVC(): AccountError {
         return new AccountError("Can't calculate deploy params: missing required TVC.");
     }
+    static missingBOC(): AccountError {
+        return new AccountError("Account has an empty BOC.");
```

Comment:
Add more informative error message

Reply:
Can we say "Account has an empty BOC,  most likely its type is "NonExist"?
Are there other situations where BOC can be empty?

Reply:
Account has an empty BOC. Possible reason is: account was deleted (has account type "NonExist")